src/style.css

```diff
@@ -0,0 +1,120 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 700;
+}
+
+.html {
+  font-family: Avenir, sans-serif;
+}
+
+.searchbar--big {
+  position: relative;
+  width: 100%;
+  background: #fff;
+  margin-top: 20px;
+}
+
+.icon {
+  position: absolute;
+}
+
+.icon--big {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.icon--small {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.input--big {
+  text-indent: 62px;
+  height: 70px;
+  width: 100%;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.searchbar--small {
+  position: relative;
+  margin-top: 20px;
+  width: 100%;
+  height: 42px;
+  background: #fff;
+}
+
+.input--small {
+  text-indent: 33px;
+  height: 42px;
+  width: 100%;
+  font-family: Avenir, sans-serif;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}

```

Comment:
You can add `input` class for both small and big inputs and move all the similar styled under that selector
![image](https://user-images.githubusercontent.com/96409563/230026086-40c7dba4-fe34-4728-b930-9b23125a0414.png)